template.json

```diff
@@ -216,14 +217,16 @@
                 "typeHandlerVersion": "2.0",
                 "autoUpgradeMinorVersion": true,
                 "settings": {
-                    "fileUris": "[split('https://raw.githubusercontent.com/theonemule/fahclient-azure-vm/master/install.sh', ' ')]",
```

Comment:
fileUris needs to be an array, but it can be an array with only one element. Easier to read with just square brackets around it, I thought.